samples/interactive-tutorials/prediction/get_prediction.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2021 Google Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# [START prediction_for_products]
+# Call Retail API to get predictions from Recommendation AI using simple request.
+#
+import os
+
+import google.auth
+from google.cloud.retail_v2 import (
+    PredictionServiceClient, UserEvent, ProductDetail,
+    Product, PredictRequest
+)
+from google.api_core.client_options import ClientOptions
+
+project_id = google.auth.default()[1]
+placement_id = os.environ["GOOGLE_CLOUD_PLACEMENT"]
+ENDPOINT = "retail.googleapis.com"
+
+
+# get prediction service client
+def get_search_service_client():
+    predict_client_options = ClientOptions(ENDPOINT)
+    return PredictionServiceClient(client_options=predict_client_options)
```

Comment:
Please remove `client_options` as it's redundant

Reply:
Fixed